packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1363,6 +1378,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         // make space expand/collapse the comboBox
         // and allow the event to propagate
         if (!allowFreeform && autoComplete === 'off') {
+          this._focusInputAfterClose = !!this.state.isOpen;
```

Comment:
Does it make sense to not call setState directly, but have a helper function that calls set state and also sets this._focusInputAfterClose?

Reply:
Yeah, good call, it also helps make it more clear when we are updating that value